### PR TITLE
WIP: Making a debug flag!!!!!!

### PR DESCRIPTION
**Pull request by pmt**
:beetle: :beetle: :beetle:

### CODE REVIEW

**Comment by jazzdan** [comment]
(you prolly wanna remove Nick's commit from your branch, and then rebase against master)

File: internal/cli/cli.go [hunk 1]
```diff
@@ -7,14 +7,18 @@ import (
...
 func Execute() {
+
 	rootCmd := &cobra.Command{
 		Use:   "tilt",
 		Short: "tilt is great, yo",
 	}
 
 	addCommand(rootCmd, &upCmd{})
 	addCommand(rootCmd, &daemonCmd{})
+	rootCmd.PersistentFlags().BoolVarP(&debug, "debug", "d", false, "Run with verbose debug messages")
```
Comment (jazzdan) [inline internal/cli/cli.go]: Did we ever figure out how to call `SetDebugMode` appropriately here?

Comment (pmt) [reply internal/cli/cli.go]: See up.go :))

Comment (jazzdan) [reply internal/cli/cli.go]: Aha, brilliant!

**Pull request closed without merging**